File: sub-web-model/src/main/java/org/github/base/service/impl/SysRoleMenuServiceImpl.java
package org.github.base.service.impl;

import org.github.base.entity.SysRoleMenuEntity;
import org.github.base.mapper.ISysRoleMenuMapper;
import org.github.base.service.ISysRoleMenuService;
import org.github.base.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 * <p>
 * 角色与菜单对应关系 服务实现类
 * </p>
 *
 * @author JYD_XL
 * @since 2021-05-12
 */
@Service
public class SysRoleMenuServiceImpl extends ServiceImpl<ISysRoleMenuMapper, SysRoleMenuEntity> implements ISysRoleMenuService {}

File: sub-mysql-mydb-service/src/main/java/org/github/mysql/mydb/base/service/impl/SysRoleMenuServiceImpl.kt
@file:Suppress("RedundantSemicolon")

package org.github.mysql.mydb.base.service.impl;

import org.github.mysql.mydb.base.entity.SysRoleMenuEntity;
import org.github.mysql.mydb.base.mapper.ISysRoleMenuMapper;
import org.github.mysql.mydb.base.service.ISysRoleMenuService;
import org.github.base.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 * 角色-菜单关系表 服务实现类
 *
 * @author JYD_XL
 */
@Service
open class SysRoleMenuServiceImpl : ServiceImpl<ISysRoleMenuMapper, SysRoleMenuEntity>(), ISysRoleMenuService
